tests/unit/workflows/nodejs_npm/test_actions.py

```diff
@@ -113,24 +113,31 @@ def test_raises_action_failed_when_npm_fails(self, SubprocessNpmMock):
 class TestNodejsNpmrcAndLockfileCopyAction(TestCase):
     @parameterized.expand(
         [
-            [False, False],
-            [True, False],
-            [False, True],
-            [True, True],
+            [False, False, False],
+            [True, False, False],
+            [False, True, False],
+            [True, True, False],
+            [False, False, True],
+            [True, False, True],
+            [False, True, True],
+            [True, True, True],
```

Comment:
nit: if this is all the combination of these 3 params, we can also write it like this;
```
@parameterized.expand(
  itertools.product([True, False], [True, False], [True, False])
)
```

Reply:
Oh cool, let me try that, thanks!